CHANGELOG.md

```diff
@@ -15,23 +15,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     * `.register_mcp_tools_http_stream_async()` and `.register_mcp_tools_stdio_async()`: for registering tools from a [MCP server](https://modelcontextprotocol.io/). (#39)
     * `.get_tools()` and `.set_tools()`: for fine-grained control over registered tools. (#39)
     * `.add_turn()`: to add `Turn`(s) to the current chat history. (#126)
+    * `Chat` gains new `.get_cost()` method, making it easier to get the estimated cost of your chat. Use our pricing list or bring your own token prices. (#106)
 * Tool functions passed to `.register_tool()` can now `yield` numerous results. (#39)
 * New content classes (`ContentToolResultImage` and `ContentToolResultResource`) were added, primarily to represent MCP tool results that include images/files. (#39)
 * A `Tool` can now be constructed from a pre-existing tool schema (via a new `__init__` method). (#39)
 * The `Chat.app()` method gains a `host` parameter. (#122) 
 * `ChatGithub()` now supports the more standard `GITHUB_TOKEN` environment variable for storing the API key. (#123)
+* Token pricing can now be looked up from our internally maintained price list using `get_token_pricing()`. (#106)
 
 ### Breaking changes
 
 * `Chat` constructors (`ChatOpenAI()`, `ChatAnthropic()`, etc) no longer have a `turns` keyword parameter. Use the `.set_turns()` method instead to set the (initial) chat history. (#126)
 
 ### Changes
 
+#### Breaking Changes
+
+* `Chat`'s `.tokens()` methods have been removed in favor of `.get_tokens()` which returns both cumulative tokens in the turn and discrete tokens. (#106)
+* `Provider`s now require a name and model. Sensible defaults are supplied for our implemented providers. They are set during initialization as properties. (#106)
+
+#### Other Changes
+
 * `Tool`'s constructor no longer takes a function as input. Use the new `.from_func()` method instead to create a `Tool` from a function. (#39)
 * `.register_tool()` now throws an exception when the tool has the same name as an already registered tool. Set the new `force` parameter to `True` to force the registration. (#39)
 
 ### Improvements
 
+* `Chat`'s representation will now include cost information if it can be calculated. (#106)
+* `token_usage()` will include cost if it can be calculated. (#106)
```

Comment:
```suggestion
* `Chat`'s representation now includes cost information if it can be calculated. (#106)
* `token_usage()` includes cost if it can be calculated. (#106)
```